app/controllers/pages_controller.rb

```diff
@@ -62,7 +62,7 @@ def get_job(pbsid, cluster)
       Jobstatusdata.new({name: name, attribs: attribs}, cluster.id, true)
 
     rescue
-      "[{\"name\":\"#{pbsid}\",\"error\":\"Job data expired or invalid.\"}]"
+      { name: pbsid, error: "INVALID" }
```

Comment:
Right now the user doesn't see the message. I only check whether an error message is set or not. This message is really only for the benefit of people snooping the network response. I think `e.message` is just going to return an error about unparseable data, so it may not be useful here.